app/src/main/java/org/oppia/app/walkthrough/topiclist/WalkthroughTopicListFragment.kt

```diff
@@ -0,0 +1,24 @@
+package org.oppia.app.walkthrough.topiclist
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+class WalkthroughTopicListFragment : InjectableFragment() {
+  @Inject
```

Comment:
Please write @Inject in the same line here and elsewhere eg: 
 `@Inject lateinit var walkthroughTopicListFragmentPresenter: WalkthroughTopicListFragmentPresenter
`

Reply:
done
[](url)